bionic/core/flow_execution.py

```diff
@@ -278,14 +278,17 @@ def _process_active_entry(self, entry):
                 remote_subgraph.get_stripped_state(target_entry.state)
                 for target_entry in target_entries
             ]
-            new_core = self._core.evolve(aip_executor=None, process_executor=None)
-            new_task_completion_runner = TaskCompletionRunner(
+            new_core = self._context.core.evolve(
```

Comment:
nit: comments on why we are removing the executors this would be nice.

Reply:
Agreed. I see you added it to one of your PRs too.